src/components/product/ProductForm.client.jsx

```diff
@@ -143,6 +144,9 @@ export function ProductForm({productData}) {
           selectedVariant={selectedVariant}
           productData={productData}
         />
+        <SocialCountSection
```

Comment:
@gouravswym will this component rerender on `add to wishlist` btn click and update the social count?